x-pack/plugins/apm/public/components/app/service_groups/service_group_save/select_services.tsx

```diff
@@ -70,6 +67,7 @@ export function SelectServices({
 }: Props) {
   const [kuery, setKuery] = useState(serviceGroup?.kuery || '');
   const [stagedKuery, setStagedKuery] = useState(serviceGroup?.kuery || '');
+  const [kueryValidationMessage, setKueryValidationMessage] = useState('');
```

Comment:
nit: seems better to use `undefined` instead of empty string

```suggestion
  const [kueryValidationMessage, setKueryValidationMessage] = useState<string | undefined>(undefined);
```